File: Semester 4/Systems for Design and Implementation/django_backend/A1/snippets/models/TravelAgency.py
from django.db import models
class TravelAgency(models.Model):
    name=models.CharField(max_length=100)
    website=models.CharField(max_length=100)
    address=models.CharField(max_length=100)
    nrOfEmployees=models.IntegerField()
    nrOfOffers=models.IntegerField()
    cityBreak=models.ManyToManyField('CityBreak',through='CityBreakAgency')

    def __str__(self):
        return self.name

File: Semester 4/Systems for Design and Implementation/react_frontend/src/models/TravelAgency.ts
import { CityBreak } from "./CityBreak";
export interface TravelAgency{
    id:number;
    name:string; 
    website:string; 
    address:string;
    nrOfEmployees:number;
    nrOfOffers:number;
    cityBreaks:CityBreak[];
}